src/main/java/edu/ucsb/ucsbcslas/controllers/OnlineOfficeHoursController.java

```diff
@@ -124,6 +124,16 @@
         return ResponseEntity.ok().body(body);
     }
 
+    @GetMapping(value = "/api/member/officeHours", produces = "application/json")
```

Comment:
Should this endpoint also have a check that returns an unauthorized status when the user is not a member?

Reply:
Good point.... I'm going to have them do that now.

Reply:
They did it :-)